docs/src/contributing/development-environment.md

```diff
@@ -151,42 +126,7 @@ The `deis` client is also written in Go. Your Deis client should match your serv
 
 ## Start Up a Development Cluster
 
-Our host system is now configured for controlling a Deis cluster. The next thing to do is begin standing up a development cluster.
-
-When developing locally, we want deisctl to check our local unit files so that any changes are reflected in our Deis cluster. The easiest way to do this is to set an environment variable telling deisctl where to look. Assuming the variable `$DEIS` points to the location if the deis source code, we want something like this:
-
-    $ export DEISCTL_UNITS=$DEIS/deisctl/units
-
-To start up and configure a local vagrant cluster for development, you can use the `dev-cluster` target.
-
-    $ make dev-cluster
-
-This may take a while to run the first time. At the end of the process, you will be prompted to run `deisctl start platform`. Hold off on that task for now. We will come back to it later.
-
-To verify that the cluster is running, you should be able to connect to the nodes on your Deis cluster:
-
-    $ vagrant status
-    Current machine states:
-
-    deis-01               running (virtualbox)
-    deis-02               running (virtualbox)
-    deis-03               running (virtualbox)
-
-    $ vagrant ssh deis-01
-    Last login: Tue Jun  2 18:26:30 2015 from 10.0.2.2
-     * *    *   *****    ddddd   eeeeeee iiiiiii   ssss
-    *   *  * *  *   *     d   d   e    e    i     s    s
-     * *  ***** *****     d    d  e         i    s
-    *****  * *    *       d     d e         i     s
-    *   * *   *  * *      d     d eee       i      sss
-    *****  * *  *****     d     d e         i         s
-      *   *****  * *      d    d  e         i          s
-     * *  *   * *   *     d   d   e    e    i    s    s
-    ***** *****  * *     ddddd   eeeeeee iiiiiii  ssss
-
-    Welcome to Deis     Powered by CoreOS
-
-With a dev cluster now running, we are ready to set up a local Docker registry.
+TODO: Document some approved "zero to kubernetes" options.
```

Comment:
So I've been resisting taking on the role of guiding people on k8s. But reading the last few comments, and especially seeing @arschles 's table, I have changed my mind. I really like the idea of adding a table like the above. (Maybe break kube-up into a few lines to stress the big back-ends it supplies.)

Maybe the last column could be "easy, medium, hard" instead of "low, medium, high".

Given this, the list I'd like to see would be:

Kube-up with Vagrant
Kube-up with GCE (maybe)
micro-kube
GKE
Something with AWS (kube-up with aws?).
DigitalOcean if anyone has done it

And then a disclaimer saying that any Kubernetes 1.1+ platform _should_ support Deis.